src/components/avatarHeader/avatarHeaderView.js

```diff
@@ -1,63 +1,48 @@
 import React from 'react';
-import { View, Text, SafeAreaView, TouchableOpacity } from 'react-native';
-import { useNavigation } from '@react-navigation/native';
-import LinearGradient from 'react-native-linear-gradient';
+import { View, Text, TouchableOpacity } from 'react-native';
 
 import { UserAvatar } from '../userAvatar';
 import { IconButton } from '../iconButton';
 
 // Styles
 import styles from './avatarHeaderStyles';
 
-const AvatarHeader =
-  ({ username, name, reputation, avatarUrl, showImageUploadActions, isUploading }) =>
-  () => {
-    const navigation = useNavigation();
+const AvatarHeader = ({
+  username,
+  name,
+  reputation,
+  avatarUrl,
+  showImageUploadActions,
+  isUploading,
+}) => {
+  return (
+    <View style={styles.headerContainer}>
+      <View style={styles.wrapper}>
+        <TouchableOpacity onPress={showImageUploadActions}>
+          <UserAvatar
+            key={`${avatarUrl}-${username}`}
+            noAction
+            size="xl"
+            username={username}
+            avatarUrl={avatarUrl}
+            isLoading={isUploading}
+          />
+          <IconButton
```

Comment:
Button background should primary blue, and perhaps use `pencil` icon instead `plus/add`, can also be slightly bigger may be.

<img width="190" alt="Screenshot 2024-04-16 at 18 09 07" src="https://github.com/ecency/ecency-mobile/assets/6298342/5f94eac6-915f-46c9-bcd6-f43a51b765f9">